### PR TITLE
FAQ item for OpenCart and WooCommerce generic gateway

### PR DESCRIPTION
**Pull request by Daniel-MultiSafepay**
This PR includes a short FAQ article for OpenCart and WooCommerce plugins on generic gateways.

The base branch is a generic gateway started by @Thomas-MultiSafepay to unite all the changes related to this concept and submit a PR to master at the right moment.

### CODE REVIEW

File: content/integrations/plugins/opencart/faq/generic-gateways.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title : "Generic gateways"
+meta_title: "Generic gateways - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+---
+The OpenCart plugin offer a generic gateway; that can be found in plugin settings page, listed in payment methods tab.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/opencart/faq/generic-gateways.md:6]:
```suggestion
The OpenCart plugin offers a generic gateway. The generic gateway can be found on the plugin settings page, listed in the payment methods tab.
```

File: content/integrations/plugins/opencart/faq/generic-gateways.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+All of our payment methods are supported. It is possible to set the gateway code, gateway logo, gateway label and whether the shopping cart should be included in refund transactions; which is required if the gateway code used is a billing suite payment method.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/opencart/faq/generic-gateways.md:8]:
```suggestion
All of our payment methods are supported. It is possible to set the gateway code, gateway logo, gateway label, and whether the shopping cart should be included in refund transactions. Specifying whether the shopping cart should be included in refund transactions is required if the gateway code used is that of a billing suite payment method.
```

File: content/integrations/plugins/opencart/faq/generic-gateways.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+In the same way than others gateways, is possible also filter the generic gateway by geo zone, currency, min amount, max amount, customer groups, and a set a custom initial order status.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/opencart/faq/generic-gateways.md:10]:
```suggestion
Similar to other gateways, it is possible to filter the generic gateway by:

* geographic zone
* currency
* min amount
* max amount
* customer groups

Furthermore, you can set a custom initial order status.
```

File: content/integrations/plugins/woocommerce/faq/generic-gateways.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title : "Generic gateways"
+meta_title: "Generic gateways - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+---
+The OpenCart plugin offer a generic gateway; that can be found in WooCommerce Settings; under payment tab, which list all payment methods.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/woocommerce/faq/generic-gateways.md:6]:
```suggestion
The OpenCart plugin offers a generic gateway. The generic gateway can be found in your WooCommerce Settings under the payment tab, which lists all payment methods.
```

File: content/integrations/plugins/woocommerce/faq/generic-gateways.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+All of our payment methods are supported. It is possible to set the gateway code, gateway logo, gateway label and whether the shopping cart should be included in refund transactions; which is required if the gateway code used is a billing suite payment method.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/woocommerce/faq/generic-gateways.md:8]:
```suggestion
All of our payment methods are supported. It is possible to set the gateway code, gateway logo, gateway label, and whether the shopping cart should be included in refund transactions. Specifying whether the shopping cart should be included in refund transactions is required if the gateway code used is that of a billing suite payment method.
```

File: content/integrations/plugins/woocommerce/faq/generic-gateways.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+In the same way than others gateways, is possible also filter the payment method by country, min amount, max amount, and a set a custom initial order status.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/woocommerce/faq/generic-gateways.md:10]:
```suggestion
Similar to other gateways, it is possible to filter the generic gateway by:

* geographic zone
* currency
* min amount
* max amount
* customer groups
Furthermore, you can set a custom initial order status.
```

File: content/integrations/plugins/opencart/faq/generic-gateways.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+Full refunding actions and backend orders are fully supported.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/opencart/faq/generic-gateways.md:12]:
```suggestion
Full refunds and backend orders are fully supported.
```

File: content/integrations/plugins/woocommerce/faq/generic-gateways.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+meta_title: "Generic gateways - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+---
+The OpenCart plugin offer a generic gateway; that can be found in WooCommerce Settings; under payment tab, which list all payment methods.
+
+All of our payment methods are supported. It is possible to set the gateway code, gateway logo, gateway label and whether the shopping cart should be included in refund transactions; which is required if the gateway code used is a billing suite payment method.
+
+In the same way than others gateways, is possible also filter the payment method by country, min amount, max amount, and a set a custom initial order status. 
+
+Full refunding actions, partial refunds action (except for billing suite payment methods), and backend orders are fully supported.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/woocommerce/faq/generic-gateways.md:12]:
```suggestion
Full refunds, partial refunds (except for billing suite payment methods), and backend orders are fully supported.
```

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]
Mostly English grammar and rephrasing suggestions.

File: content/integrations/plugins/woocommerce/faq/generic-gateways.md (hunk 4 above)
Comment (Vinod-MultiSafepay) [inline content/integrations/plugins/woocommerce/faq/generic-gateways.md:6]: I think OpenCart here should be WooCommerce

**Review by Vinod-MultiSafepay — CHANGES_REQUESTED** [review]
Small feedback

File: content/integrations/plugins/woocommerce/faq/generic-gateways.md (hunk 4 above)
Comment (Daniel-MultiSafepay) [reply content/integrations/plugins/woocommerce/faq/generic-gateways.md]: Thanks Vinod. ! Is done now.

**Review by Vinod-MultiSafepay — APPROVED** [review]

**Pull request merged**